### ISSUE TITLE
The game crashes at typing the focus bag button

### ISSUE THREAD

**Issue by Ronanfalcon** [Polarice3/Goety#1]
Like the title says, I created a new world and, I was not aware of the focus bag key binding at first, and as soon as the map loaded and I was ready to playing, I used the magnesium mod's zoom function randomly, which is assigned to 'c' key too, and "discovered" it, as the game crashes because of it.

Nothing happens when the focus bag is with me, and it opens normally.

So I tried to change the bindings, in any way possible, but it continues to crash, if it is not with me, regardless of the button combination I tried.

The crash report is here: https://pastebin.com/H7nBK7qj

I don't have knowledge in coding and anything, so I'm posting this here, as I didn't find clue about this error anywhere

**Comment by Polarice3** [comment]
Ah! Sorry about that! I messed up on coding that part! Just Fixed it with this new Update! Now you should be able to press the focus bag key without crashing! Pls inform me if the problem still persists!

Here's the new Update: https://www.curseforge.com/minecraft/mc-mods/goety/files/3758820

Thank you!

**Comment by Ronanfalcon** [comment]
Yes, indeed it's fixed... at first...

However, something else happened.

After testing, and seeing that it was fixed, I saved/closed the map, but immediately java reported a heapdump.
And then it started to send heapdumps every time I tried to start it again, still with Goety.
All this happened with 2gb of allocated memory, so I tried 3gb after that, and the game started, but java sent the heapdump when trying to create the map.
Lastly, I just removed Goety, and tried with 2gb allocated again (as it was from the start), but again, java kept sending the heapdump - i.e. my game corrupted on that, although I have no idea what that is.

I'll have delete the game, and download it again to see if that fixes it (I believe so), but I can't say anything else, because there's no crash report, or log to show you (as I had to shutdown the game by force every time).

I don't really know if has something to do with your mod, but I hope this helps.
